### ISSUE TITLE
Page to Blog macro not working

### ISSUE THREAD

**Issue by kberzinch** [chris-kent/Confluence-Blog-Tools#43]
We are running Confluence Server 6.14.0 and plugin version 3.19.

The macro is embedded in a space template, but saving the template to a new page doesn't create a blog post, just creates a new page.

I have noticed that the "Publish Blog" button appears momentarily. but then is replaced by the usual "Save" button when the page finishes loading. The JavaScript console also has the following messages in sequence.

```
License OK
Create Form
Page Template or Page Draft
Converting this Page to BLOGPOST
```

**Comment by chris-kent** [comment]
Hi,

I have seen the problem, something new Atlassian have added in 6.14.

I have removed compatibility for this plugin v 3.19 with this latest
version of Confluence.

I'll have an update soon.

Cheers

**Comment by kberzinch** [comment]
Looks to be fixed in 3.21. Thanks!
